### PR TITLE
My fix issues76n80 branch

### PR DESCRIPTION
**Pull request by zabela**
Small changes to the master like removing gulp from the contributing file as it is not used and adding small fixes for issues #76 & #80:
- update package & package-lock.json files
- changed the Firebaseui import in src/Auth.js

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,11 +21,12 @@
     "eslint-config-google": "^0.9.1",
     "firebase-tools": "^7.2.2",
     "glob-all": "^3.1.0",
+    "har-validator": "^5.1.3",
```
Comment (jhuleatt) [inline package.json]: why was this added? please remove if it was added by accident

Comment (zabela) [reply package.json]: That was no accident but as fix as suggested in issue #66 by @r0b78

**Comment by samtstern** [comment]
We're going to be deprecating the friendlypix-web repo so we're closing all open PRs and Issues.  Thank you for you contribution!

For new web developers we recommend the [Get to know Firebase for Web](https://codelabs.developers.google.com/codelabs/firebase-get-to-know-web) codelab.
